v2/samples/dataprotection/v1api/v1api20230101_backupvaultsbackupinstance.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: dataprotection.azure.com/v1api20230101
+kind: BackupVaultsBackupInstance
+metadata:
+  name: asotestbackupinstance
+  namespace: default
+spec:
+  owner:
+    name: asotestbackupvault
+  properties:
+    objectType: BackupInstance
+    friendlyName: asotestbackupinstance
+    dataSourceInfo:
+      objectType: Datasource
+      datasourceType: Microsoft.ContainerService/managedClusters
+      resourceUri: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.ContainerService/managedClusters/aks
```

Comment:
I'd suggest replacing `resourceGroups/rg` with `resourceGroups/aso-sample-rg` here and below in armIDs.

Reply:
done the changes